src/main/scala/ch/openolitor/stammdaten/repositories/StammdatenRepositoryQueries.scala

```diff
@@ -801,20 +801,29 @@ trait StammdatenRepositoryQueries extends LazyLogging with StammdatenDBMappings
     withSQL {
       select
         .from(depotlieferungAboMapping as depotlieferungAbo)
-        .leftJoin(abwesenheitMapping as abwesenheit).on(depotlieferungAbo.id, abwesenheit.aboId)
-        .leftJoin(abotypMapping as aboTyp).on(depotlieferungAbo.abotypId, aboTyp.id)
-        .leftJoin(vertriebMapping as vertrieb).on(depotlieferungAbo.vertriebId, vertrieb.id)
-        .leftJoin(lieferungMapping as lieferung).on(
-          sqls.eq(vertrieb.id, lieferung.vertriebId)
-            .and.eq(lieferung.abotypId, depotlieferungAbo.abotypId)
-        )
+        .leftJoin(lieferungMapping as lieferung).on(lieferung.abotypId, depotlieferungAbo.abotypId)
         .leftJoin(lieferplanungMapping as lieferplanung).on(
           sqls.eq(lieferung.lieferplanungId, lieferplanung.id).and(sqls.toAndConditionOpt(
             ausstehend map (_ => sqls.isNull(lieferung.lieferplanungId).or.eq(lieferplanung.status, Ungeplant)
               .or.eq(lieferplanung.status, Offen)
               .or.eq(depotlieferungAbo.aktiv, false))
           ))
         )
+        .leftJoin(abwesenheitMapping as abwesenheit).on(
+          sqls.eq(depotlieferungAbo.id, abwesenheit.aboId).and(sqls.toAndConditionOpt(
+            ausstehend map (_ => sqls.isNull(lieferung.lieferplanungId).or.eq(lieferplanung.status, Ungeplant)
```

Comment:
Can we still see the past Abwesenheiten in the Admin-Portal like this?

Reply:
yes, but this only filters the open and unplan ones. However, this needs to be seriuosly test